website/index.html

```diff
@@ -9,7 +9,7 @@
 	<link rel="stylesheet" href="style.css">
     </head>
     <body>
-      <img src="stratis_CMYK_grey.jpg" width=338 height=205 />
+      <img src="stratis_CMYK_grey.jpg" width=338 height=205 alt="Stratis logo" />
```

Comment:
Tidy isn't satisfied unless you put quote marks around height and width values.

Reply:
I was using: https://validator.w3.org/